packages/api/models/index.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const sequelize = require('../helper/sequelize.js');
+const basename = path.basename(__filename);
+const db = {};
+
+// Instantiate all of the models defined in this directory
+// Then store them on the db object
+fs
+  .readdirSync(__dirname)
+  .filter(file => {
+    return (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js');
+  })
+  .forEach(file => {
+    const model = require(path.join(__dirname, file))(sequelize, Sequelize.DataTypes);
+    db[model.name] = model;
+  });
```

Comment:
I prefer `.reduce` to `.forEach` syntax as I find it is cleaner, but possibly harder to understand at first glance. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

This would return directly to the variable instead of setting it and changing the values later: 

```javascript
const db = fs
// snip readdirsync and filter functions
// added bonus that filter returns an array that can be used in this chain. 
  .reduce((db, file) => {
    const model = require(path.join(__dirname, file))(sequelize, Sequelize.DataTypes);
    db[model.name] = model;
  }, {})
```

This might not be needed for this but just wanted to show a different way of doing it.